cli/cmd/top.go

```diff
@@ -175,7 +342,7 @@ func getTrafficByResourceFromAPI(w io.Writer, client pb.ApiClient, req *pb.TapBy
 	go recvEvents(rsp, requestCh, done)
 	go pollInput(done)
 
-	renderTable(requestCh, done, !options.hideSources)
+	renderTable(requestCh, done)
```

Comment:
unrelated to this PR, but just realized `options` is no longer needed as a param to this function.